src/ports/rentals/components.ts

```diff
@@ -207,8 +212,71 @@ export function createRentalsComponent(
     return parsedResult
   }
 
+  async function getRentalAssets(
+    filters: GetRentalAssetFilters
+  ): Promise<RentalAsset[]> {
+    function getSubgraphQueryOptions() {
+      const queryOptions: string[] = []
+
+      const where: string[] = []
+
+      function mapListToFilter(xs: string[] | undefined, name: string) {
```

Comment:
Quick question! What does `xs` stands for?

Reply:
Removed the function in favor of using always _in